src/components/search/input.tsx

```diff
@@ -133,7 +133,7 @@ const SearchBox = ({ refine, onFocus, currentRefinement, isOpened, ...rest }: an
   const timeoutId = React.useRef(null)
   const inputEl = React.useRef(null)
   const { width } = useWindowDimensions()
-  const placeholderText = width > 640 ? 'Search Documentation...' : 'Search'
+  const placeholderText = width > 400 ? 'Search Documentation...' : 'Search'
```

Comment:
if you really wanna need only one as always then just put as any one of it, since if you reference #991, i think it only suggest to have one all the time:
```suggestion
  const placeholderText = 'Search Documentation...'
```
or 

```suggestion
  const placeholderText = 'Search'
```

Reply:
THanks for your comment @rahul3v . We'll try to make it work before reverting to one of the two.